src/main/java/qna/domain/User.java

```diff
@@ -81,24 +82,31 @@ public Long getId() {
         return id;
     }
 
-    public String getUserId() {
+    public UserId getUserId() {
         return userId;
     }
 
-    public String getPassword() {
-        return password;
+    public void changeName(String name) {
```

Comment:
```suggestion
    public void rename(String name) {
```

단순 의견입니다.
rename은 어때요?

Reply:
change와 re-의 차이를 못찾겠네요. 제안주신 이유 알수 있을까요?!

Reply:
rename은 사전에도 등재된 단어다 보니 영미권에서 더 자연스럽게 쓰이는 것 같아 제안드렸습니다.
인텔리제이같은 IDE 에서도 보면 이런 기능의 명칭은 rename 으로 되어있구요.